UIKit/Tasks/CustomLabels/ChoiceLabel.swift

```diff
@@ -0,0 +1,16 @@
+// ChoiceLabel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Класс для кастомного лейбла для избежания копи паста
```

Comment:
```suggestion
/// Класс для кастомного лейбла для избежания копи паста
```
не по гайду название